src/muse/decisions.py

```diff
@@ -92,12 +92,12 @@ def decorated(*args, **kwargs) -> DataArray:
     return decorated
 
 
-def coeff_sign(domin: bool, coeff: Any):
+def coeff_sign(domain: bool, coeff: Any):
     """Adds sign to coefficient depending on minimizing or maximizing.
 
     This function standardizes across the decision methods.
     """
-    return coeff if domin else -coeff
+    return coeff if domain else -coeff
```

Comment:
Codespell flagged this... @dalonsoa Am I right that this is supposed to be "domain"?

Reply:
Ohhhh that makes sense. I should have read the docstring more closely!